src/consts.ts

```diff
@@ -1,4 +1,12 @@
 const MAILTO_URL = 'mailto:hello@sendchinatownlove.com';
+const emailIcon = require('./images/social-icons/email.png');
+const instagramIcon = require('./images/social-icons/instagram.png');
+const tiktokIcon = require('./images/social-icons/tiktok.png');
+const facebookIcon = require('./images/social-icons/facebook.png');
+const twitterIcon = require('./images/social-icons/twitter.png');
+const linkedinIcon = require('./images/social-icons/linkedin.png');
+const youtubeIcon = require('./images/social-icons/youtube.png');
+const wechatIcon = require('./images/social-icons/weixin.png');
```

Comment:
You should be able to use import syntax on anything that's not a .js/.ts/.tsx file like in the [Hero Banner](https://github.com/sendchinatownlove/sendchinatownlove.github.io/blob/6a63221872407f5a36df84ac6f7eca3081ecec9c/src/components/HeroBanner/HeroBanner.tsx#L3) code

Reply:
+1, looks weird and is equivalent, but we generally prefer the `import X from X` syntax instead of the `const ... require()`